src/components/bouquets/BouquetDatasetList.vue

```diff
@@ -197,8 +233,9 @@ const triggerReorder = () => {
             <!-- eslint-disable-next-line vue/no-v-html -->
             <div v-html="fromMarkdown(dataset.purpose)"></div>
             <BouquetDatasetCard
-              v-if="dataset.id"
+              v-if="dataset.id && datasetsContent?.length"
```

Comment:
```suggestion
              v-if="dataset.id && datasetsContent && datasetsContent[index]"
```

`.length` ne me parait pas utile et le check sur l'index est nécessaire pour éviter un warning en console.